R/timeseriessummary-data.R

```diff
@@ -388,4 +389,86 @@ parseTSSGapTolerances <- function(reportData, timezone){
   }
   
   return(gapTolerances)
-}
\ No newline at end of file
+}
+
+#' Format TSS Correction Parameters
+#' @description formats the parameters so they are prettier to print
+#' @param corrections the corrections for the report
+#' @param timezone the timezone to parse data into
+#' @return corrections with formattedParameters field to print in report
+formatCorrectionParameters <- function(corrections, timezone) {
+  
+  corrections[["formattedParameters"]] <- ""
+  
+  if(!isEmptyOrBlank(corrections) && !isEmptyVar(corrections)) {
+    
+    for (i in seq(nrow(corrections))) {
```

Comment:
then we can write a few simple tests for each one of those - sorry, I know this creates a lot more work. If you don't think it's really adding anything, don't worry about it. I just get intimated by giant if-else kind of functions